langchain4j-vertex-ai-gemini/src/test/java/dev/langchain4j/model/vertexai/VertexAiGeminiChatModelIT.java

```diff
@@ -213,38 +221,85 @@ void should_accept_text_and_multiple_base64_images() {
         String catBase64Data = Base64.getEncoder().encodeToString(readBytes(CAT_IMAGE_URL));
         String diceBase64Data = Base64.getEncoder().encodeToString(readBytes(DICE_IMAGE_URL));
         UserMessage userMessage = UserMessage.from(
-                ImageContent.from(catBase64Data, "image/png"),
-                ImageContent.from(diceBase64Data, "image/png"),
-                TextContent.from("What do you see? Reply with one word per image.")
+            ImageContent.from(catBase64Data, "image/png"),
+            ImageContent.from(diceBase64Data, "image/png"),
+            TextContent.from("What do you see? Reply with one word per image.")
         );
 
         // when
         Response<AiMessage> response = visionModel.generate(userMessage);
 
         // then
         assertThat(response.content().text())
-                .containsIgnoringCase("cat")
-                .containsIgnoringCase("dice");
+            .containsIgnoringCase("cat")
+            .containsIgnoringCase("dice");
     }
 
     @Test
     void should_accept_text_and_multiple_images_from_different_sources() {
 
         // given
         UserMessage userMessage = UserMessage.from(
-                ImageContent.from(CAT_IMAGE_URL),
-                ImageContent.from("gs://langchain4j-test/dog.jpg"),
-                ImageContent.from(Base64.getEncoder().encodeToString(readBytes(DICE_IMAGE_URL)), "image/png"),
-                TextContent.from("What do you see? Reply with one word per image.")
+            ImageContent.from(CAT_IMAGE_URL),
+            ImageContent.from("gs://langchain4j-test/dog.jpg"),
+            ImageContent.from(Base64.getEncoder().encodeToString(readBytes(DICE_IMAGE_URL)), "image/png"),
+            TextContent.from("What do you see? Reply with one word per image.")
         );
 
         // when
         Response<AiMessage> response = visionModel.generate(userMessage);
 
         // then
         assertThat(response.content().text())
-                .containsIgnoringCase("cat")
-                .containsIgnoringCase("dog")
-                .containsIgnoringCase("dice");
+            .containsIgnoringCase("cat")
+            .containsIgnoringCase("dog")
+            .containsIgnoringCase("dice");
+    }
+
+    @Test
+    void should_accept_tools_for_function_calling() {
```

Comment:
minor: I plan to have a unified set of standartized tests for models the same way as `EmbeddingStoreIT` soon. Could you please copy/paste here tool-related tests from `OpenAiChatModelIT`?